docs/howto/performance.md

```diff
@@ -1,5 +1,5 @@
 ---
-id: advanced_performance
+id: howto_performance
```

Comment:
This one doesn't really fit "how to", I suggest we move it to the end of "Overview"